content/en/docs/developer_guide/flows/flow_triggers/oas_flow_trigger.md

```diff
@@ -54,7 +54,33 @@ If all your designed paths begin with a common prefix, i.e. `/service`, then by
 The OpenAPI specification is bound to `/apidoc/swagger.json`, `/apidoc/swagger.yaml` (both for legacy purposes), as well as `/apidoc/openapi.json` and `/apidoc/openapi.yaml`. On startup, users will see only one path in the log. For OpenAPI 2.0, it is `/apidoc/swagger.json`, otherwise, it is `/apidoc/openapi.json`.
 The prefix (`/apidoc`) is configured by changing [`apidoc.prefix` in configuration](/docs/developer_guide/project/configuration/project_configuration#apidoc).
 
-## Currently unsupported features
+<!-- ## Request handling
+Describe what we do on an inbound request before hitting the flow. Write about what validation, massaging, coercing, response codes to expect and when. -->
+
+## Response handling
+
+{{% variables/apibuilder_prod_name %}} and the OpenAPI flow-trigger do additional processing after a flow completes, before sending the response. This section describes the areas that are processed.
+
+## Response body
```

Comment:
**issue**: I can't quite follow the headings.  Double-check that these are correct.  You currently have "Response body" at the same level as "Response handling" (h2), and "Response headers" at h3.  I think this should be h3.